source/YearlyOverview/YearlyStyles.css

```diff
@@ -50,11 +50,6 @@ body {
 
 #index > a {
     font-size: 3.5vh;
-    text-decoration: none;
-}
```

Comment:
I think we should keep the text decoration as it would be helpful for the user to see